new_platforms/opteesim/Trusted/cyres_sim.c

```diff
@@ -82,7 +82,7 @@ RIOT_X509_TBS_DATA x509_ta_tbs_data = {{0},
 /* Generate a simulated identity key pair, derived from a mock FW measurement.
  */
 static void gen_sim_id(
-    const uint8_t* digest,
+    uint8_t* digest,
```

Comment:
Why was const dropped here, does it actually write to the digest?   If not, and it's just to prevent a compiler warning, then cast in line 98 etc instead and keep const here.  (And in the future the Riot APIs ought to have const in that case.)

Reply:
RiotCrypt_Hash writes into digest.